tasks/clean.js

```diff
@@ -13,8 +13,7 @@ module.exports = (gulp, dir, config, cleanAll) => () => {
 
   var cleanDist = function (directory) {
     return del([
-      path.join(directory, '**', '*'),
-      '!' + path.join(directory, 'index.html')
+      path.join(directory, '**', '*')
```

Comment:
@ryanemerson I think this step is necessary to remove index.html on clean. Please confirm